spiffs_file_preprocessor/bin_processor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "fileUtils.h"
+
+#include "bin_processor.h"
+
+void process_bin(const char * infile, const char * outdir) 
```

Comment:
Depending on final asset sizes, this may be a good candidate for heatshrink compression.